source/library/element-stub.js

```diff
@@ -0,0 +1,67 @@
+export default class ElementStub {
+	/**
+	 * Creates a new ElementStub instance. See {@link ElementStub#constructor}
+	 * @constructor
+	 * @private
+	 */
+	static create(...args) {
+		return new ElementStub(...args);
+	}
+
+	/**
+	 * Style proxy object
+	 * @type {Object}
+	 * @private
+	 */
+	style = {};
+
+	/**
+	 * Reference to currently active animationFrame
+	 * @type {Object}
+	 * @private
+	 */
+	frame = null;
+
+	/**
+	 * Creates a new ElementStub instance
+	 * @constructor
+	 * @param  {Node} element Node to stub
+	 * @param  {Window} [window=global.window] Global context to use
+	 * @return {ElementStub}         ElementStub instance
+	 * @private
+	 */
+	constructor(element, window = global.window) {
+		this.element = element;
+
+		const loop = () => {
+			this.render(this.element, this.style._style || this.style);
+			this.frame = window.requestAnimationFrame(loop);
+		};
+
+		loop();
+	}
+
+	/**
+	 * Render callback applied to each element on each frame.
+	 * @param {HTMLElement} el - element to apply the styles to
+	 * @param {object} styles - style object to apply to el
+	 * @return {ElementStub} JogWheel instance
+	 * @private
+	 */
+	render(el, styles) {
+		Object.keys(styles)
+			.filter(propertyName => propertyName !== 'length')
+			.forEach(propertyName => el.style[propertyName] = styles[propertyName]);
+		return this;
```

Comment:
Having both `.filter` to discard the `length` property and also `.forEach` to iterate could potentially be a bottleneck, especially since this method might be considered 'hot'.
Maybe it's worth investigating to use a regular for-/for-in loop or similar to get rid of those two loops.